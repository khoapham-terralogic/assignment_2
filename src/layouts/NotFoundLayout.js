import React from 'react';
import { useHistory } from 'react-router-dom';

const NotFoundLayout = () => {
    const history = useHistory()
    const goBack = () => { history.push("/") }
    return (
        <section className="notfound-layout">
            <div className="notfound-layout__container">
                404 not found
            <button onClick={goBack} className="btn">Back to home page</button>
            </div>
        </section>
    );
}

export default NotFoundLayout;
