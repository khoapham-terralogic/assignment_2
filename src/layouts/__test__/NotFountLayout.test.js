import React from 'react'
import NotFoundLayout from '../NotFoundLayout'
import { shallow } from 'enzyme';

describe('Test <NotFoundLayout />', () => {
    it('should render', () => {
        const wrapper = shallow(<NotFoundLayout />)
        expect(wrapper).toMatchSnapshot()
    });

})
