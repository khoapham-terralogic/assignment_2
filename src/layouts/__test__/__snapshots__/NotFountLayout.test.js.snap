// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test <NotFoundLayout /> should render 1`] = `
<section
  className="notfound-layout"
>
  <div
    className="notfound-layout__container"
  >
    404 not found
    <button
      className="btn"
      onClick={[Function]}
    >
      Back to home page
    </button>
  </div>
</section>
`;
